src/overview/reducer.js

```diff
@@ -28,6 +28,15 @@ const defaultState = {
     },
     showFilter: false,
     showOnlyBookmarks: false,
+    pageIdForTag: '',
+    newTag: '',
```

Comment:
What is the difference between `newTag` and `tagSearchValue` now? Can `tagSearchValue` (state of the search input) be used instead?